tests/common/gcp_type/firewall_rule_test.py

```diff
@@ -815,6 +816,25 @@ def test_firewall_rule_is_equivalent(
         rule_2 = firewall_rule.FirewallRule(**rule_2_dict)
         self.assertEqual(expected, rule_1.is_equivalent(rule_2))
 
+    def test_load_firewall_rule(self):
```

Comment:
Please add some comments here so someone else can troubleshoot this test if it fails in the future.